recipes-kernel/linux/linux-adi/0001-sc59x-fastautomdix-support.patch

```diff
@@ -0,0 +1,98 @@
+diff --git a/arch/arm/boot/dts/sc594-som-ezkit.dts b/arch/arm/boot/dts/sc594-som-ezkit.dts
+index 25acefc408c2..fd3d43c8eaa6 100644
+--- a/arch/arm/boot/dts/sc594-som-ezkit.dts
++++ b/arch/arm/boot/dts/sc594-som-ezkit.dts
+@@ -51,24 +51,22 @@
+ 			              <&ssw0 4 GPIO_ACTIVE_LOW>;	/* SPI2D2_D3_EN */
+ 		};
+ 
+-/* The button GPIO conflicts with OSPI0_D4 for this
++/* The button GPIO conflicts with OSPI0_D4 for this */
+ 		button0: button@0 {
```

Comment:
Also add a comment we using this for what . Can add the formula to calculate GPIO number in sysfs also here for reference so that we will have the information if required later.

Reply:
Added the required comment. Calculation we may not need to add her because the value we add here is from SC594 SOM schematic without any calculation.